$MirOS$

	winsock2.h must come before <sys/types.h> on mirbsd
	and <stdlib.h> on everything else: FD_ZERO & friends

--- dlls/inetcomm/internettransport.c.orig	Tue Jun 17 14:07:08 2008
+++ dlls/inetcomm/internettransport.c	Tue Jul 29 19:43:53 2008
@@ -20,13 +20,14 @@
 
 #define COBJMACROS
 
+#include "winsock2.h"
+
 #include <stdarg.h>
 #include <stdio.h>
 
 #include "windef.h"
 #include "winbase.h"
 #include "winnt.h"
-#include "winsock2.h"
 #include "ws2tcpip.h"
 
 #include "wine/debug.h"
